### ISSUE TITLE
Alternatives are the same

### ISSUE THREAD

**Issue by coderobe** [vmavromatis/absolutely-proprietary#26]
![screenshot from 2017-12-20 22-02-47](https://user-images.githubusercontent.com/8442384/34228640-a3cd3838-e5d1-11e7-9476-ca5d743b8ab9.png)
The alternatives recommend the very same packages for some of the packages, this is probably related to https://github.com/vmavromatis/absolutely-proprietary/issues/11

**Comment by vmavromatis** [comment]
Yes I noticed it too, you got me to it. This is all coming from blacklist.txt itself:
> p7zip:p7zip:::[semifree] contains nonfree unrar

A workaround would be to check if name is the same and in that case, hide it.
